sonolark/go.mod

```diff
@@ -18,51 +18,63 @@ require (
 	github.com/vmware-tanzu/sonobuoy-plugins/plugin-helper v0.0.0-20220201185710-2cdf83c452df
 	gopkg.in/yaml.v2 v2.4.0
 	istio.io/client-go v1.9.0
-	k8s.io/api v0.22.1
+	k8s.io/api v0.24.2
```

Comment:
Were these version bumps required for this specific change?

If we can separate the image bumps from the logic changes it would be preferable.

Reply:
Separated out into another PR

https://github.com/vmware-tanzu/sonobuoy-plugins/pull/171